_posts/2025-01-29-sdg23-writer.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Technical Writer for Streamlining MDAnalysis Docs and Resources
+---
+<img
+src="{{site.images}}/numfocus.png"
+title="NumFOCUS Foundation" alt="NumFOCUS Foundation"
+style="float: right; width: 10em;" />
+
+Thanks to the support of a [Small Development Grant](https://numfocus.org/programs/small-development-grants) awarded to MDAnalysis by [NumFOCUS](https://numfocus.org/), MDAnalysis is excited to start working with Namir Oues (@namiroues) as the technical writer for the funded project, "Unified and comprehensive documentation and learning resources for MDAnalysis".
```

Comment:
Add a sentence that summarizes the goal, e.g.,

..., with the goal to make it easier for new users to find the right resources to be productive with MDAnalysis.

Reply:
```suggestion
Thanks to the support of a [Small Development Grant](https://numfocus.org/programs/small-development-grants) awarded to MDAnalysis by [NumFOCUS](https://numfocus.org/), MDAnalysis is excited to start working with Namir Oues (@namiroues) as the *Technical Writer* for the funded project, "Unified and comprehensive documentation and learning resources for MDAnalysis", with the goal to make it easier for new users to find the right resources to be productive with MDAnalysis.
```
Also capitalized and highlight the position title.